### ISSUE TITLE
Visemes periodically stop responding to user voice

### ISSUE THREAD

**Issue by shiftyscales** [Frooxius/NeosPublic#1135]
Reported by Rukio, and Princess, occasionally visemes will stop functioning.

In a log from Princess was the following exception:
```
12:29:29 PM.023 ( 20 FPS)    Exception running AsyncDecode on a stream: Element: ID36D9300, Type: FrooxEngine.MultiValueStream1[System.Single], World: MMC𝕍𝕚𝕓𝕖ℤ ℍℚ, IsRemoved: False
Element: ID2B00, Type: FrooxEngine.User, World: MMC𝕍𝕚𝕓𝕖ℤ ℍℚ, IsRemoved: False
Element: ID300, Type: FrooxEngine.UserBag, World: MMC𝕍𝕚𝕓𝕖ℤ ℍℚ, IsRemoved: False, Name: Users, Disposed: False
Element: ID0, Type: FrooxEngine.World, World: MMC𝕍𝕚𝕓𝕖ℤ ℍℚ, IsRemoved: False

StreamID: ID2E3B500
System.NotSupportedException: Specified method is not supported.
  at FrooxEngine.Stream.InternalAsyncDecode (BaseX.BitBinaryReaderX reader, FrooxEngine.StreamMessage message) [0x00000] in <c0b323f9dcf94fad96116b8edd552530>:0 
  at FrooxEngine.Stream.AsyncDecode (BaseX.BitBinaryReaderX reader, FrooxEngine.StreamMessage message) [0x000c5] in <c0b323f9dcf94fad96116b8edd552530>:0 
  at FrooxEngine.SyncController.AsyncStreamDecode (FrooxEngine.StreamMessage message) [0x000c3] in <c0b323f9dcf94fad96116b8edd552530>:0 
```

I presently am unaware of any replication steps- but from the sounds of things, it's just something that periodically happens for them. I will add more detail if I discover it if there's not enough to go off of here.

**Comment by Frooxius** [comment]
Added a fix for the exception in 2020.10.12.499, please let me know if it still occurs.

**Comment by shiftyscales** [comment]
I'll reopen this issue if I hear of any additional cases of it occurring. Thanks!
